### DISCUSSION TITLE
e-puck_bluetooth.dll error

### DISCUSSION THREAD

**Discussion by kima3129** [cyberbotics/webots#6382] [General]
Hello, I have built a world in Webots and am trying to run the controller on my e-puck robot that I have written in Python. However, I get this error: 
INFO: epuck_state_machine: Starting controller: python.exe -u epuck_state_machine.py
Error: The specified module could not be found.
 (dynamic library)
Error: C:/Program Files/Webots/projects/robots/gctronic/e-puck/plugins/remote_controls/e-puck_bluetooth/e-puck_bluetooth.dll remote control library initialisation failed
Error: Cannot load the "C:/Program Files/Webots/projects/robots/gctronic/e-puck/plugins/remote_controls/e-puck_bluetooth/e-puck_bluetooth.dll" remote control library.

What could have gone wrong? I have checked that python is in my PATH and I am using python 3.11. I have also checked to see that the path "C:/Program Files/Webots/projects/robots/gctronic/e-puck/plugins/remote_controls/e-puck_bluetooth/e-puck_bluetooth.dll" exists. As a last resort, I uninstalled then reinstalled Webots only for it to automatically pull up the same world and give me the same console output.

**Comment by omichel** [comment]
There is probably some missing path in your `PATH` environment variable, preventing the `e-puck_bluetooth.dll` library to load.
Alternatively, you may simply change the [window](https://github.com/cyberbotics/webots/blob/8aba6eaae76989facf3442305c8089d3cc366bcf/projects/robots/gctronic/e-puck/protos/E-puck.proto#L21) field of your `E-puck` robot to `<generic>` or `<none>`, so that [the remote control library won't be loaded](https://github.com/cyberbotics/webots/blob/8aba6eaae76989facf3442305c8089d3cc366bcf/projects/robots/gctronic/e-puck/protos/E-puck.proto#L1188-L1194).
